src/replica/replica_stub.cpp

```diff
@@ -1754,13 +1500,10 @@ void replica_stub::on_gc()
             replica_gc.status = rep->status();
             replica_gc.plog = rep->private_log();
             replica_gc.last_durable_decree = rep->last_durable_decree();
-            replica_gc.init_offset_in_shared_log =
-                rep->get_app()->init_info().init_offset_in_shared_log;
         }
     }
 
     LOG_INFO("start to garbage collection, replica_count = {}", replica_gc_map.size());
```

Comment:
Now  that garbage collection would be removed, does this logging message need to be changed to something like "start to collect statistics", or just removed?

There is also an ending message at the end of the function:
```
    LOG_INFO("finish to garbage collection, time_used_ns = {}", dsn_now_ns() - start);
```